jcl/src/java.base/share/classes/jdk/internal/ref/CleanerShutdown.java

```diff
@@ -47,13 +47,14 @@ public static void shutdownCleaner() {
 			}
 		}
 
+		/*[IF JAVA_SPEC_VERSION < 24]*/
```

Comment:
Please remove the redundant import of `CleanerFactory` (it's in the same package) and make other imports conditional as appropriate.

Please remove the unnecessary second arguments in the calls to `getDeclaredMethod()` and `invoke()`.